### PR TITLE
x11-misc/shutter: version 0.94.3_p20200621

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

File: x11-misc/shutter/shutter-0.95_pre1.ebuild [hunk 1]
```diff
@@ -3,22 +3,23 @@
 
 EAPI=7
 
-inherit desktop eutils xdg-utils
+inherit desktop eutils vcs-snapshot xdg-utils
```
Comment (a17r) [inline x11-misc/shutter/shutter-0.95_pre1.ebuild:6]: Per convention we use `COMMIT=af237e7f415a5fa20402703490c6190dc1af9b20` here and set `S="${WORKDIR}/${PN}-${COMMIT}"` below, then don't need to inherit `vcs-snapshot` as github gives us a predictable directory name.

File: x11-misc/shutter/shutter-0.95_pre1.ebuild [hunk 2]
```diff
@@ -3,22 +3,23 @@
...
 
 DESCRIPTION="Feature-rich screenshot program"
 HOMEPAGE="https://shutter-project.org/"
-SRC_URI="https://launchpad.net/shutter/0.9x/${PV}/+download/${P}.tar.gz"
+SRC_URI="https://github.com/shutter-project/shutter/archive/af237e7f415a5fa20402703490c6190dc1af9b20.tar.gz -> ${P}.tar.gz"
```
Comment (a17r) [inline x11-misc/shutter/shutter-0.95_pre1.ebuild:10]: use `${COMMIT}` here

File: x11-misc/shutter/shutter-0.95_pre1.ebuild (hunk 1 above)
Comment (DarthGandalf) [reply x11-misc/shutter/shutter-0.95_pre1.ebuild:6]: Okay... It's shorter this way though, and is as understandable :(

**Comment by DarthGandalf** [comment]
All done

**Pull request closed without merging**